### PR TITLE
Update settings-policies-reference.md

### PR DESCRIPTION
**Pull request by NancyForero**
Change to let users know about the SIP Guest Join feature release.

### CODE REVIEW

File: Teams/settings-policies-reference.md [hunk 1]
```diff
@@ -318,6 +318,9 @@ Meeting policies are used to control what features are available in meetings org
 |Who is in the report|Everyone, but participants can opt out|This setting controls whether participants in the meeting can opt in or out of offering their attendance information in the Attendance Report.|
 |Attendance summary|Show everything|This setting controls whether to show attendance time information - such as join times, leave times, and in-meeting duration - for each meeting participant.|
 
+> [!NOTE]
+> A guest can be a participant joining from a non-Teams application or device. All the guest settings applied by your organization will apply for these participants.
+>
```
Comment (JohanFreelancer9) [inline Teams/settings-policies-reference.md:323]:
```suggestion
> A guest can be a participant joining from a non-Teams application or device. All the guest settings applied by your organization will apply to these participants.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NancyForero, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@mkbond007 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@mkbond007
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**